service/pure/src/pure.rs

```diff
@@ -170,6 +182,38 @@ pub fn get_from(hash: String, external_multiaddr: String) -> IpfsGetResult {
     }
 }
 
+#[marine]
+pub fn dag_get(hash: String) -> IpfsGetResult {
```

Comment:
why do you log put and get_from, but not get?

Reply:
Probably, we should remove logging at all. It happened because of copy-paste from previous put and get_from